.gitignore

```
@@ -0,0 +1,2 @@
+.idea/
+development/**/volume/
```

```suggestion
development/**/volume/

```

No newline eof

---

Yes. I was debating doing something like this instead:

```suggestion
development/*
```
And moving the docker-compose.yml in the root of the project